assets/test/specs/enhancedTables.spec.js

```diff
@@ -0,0 +1,51 @@
+require('jquery');
+
+describe('Given I have an enhanced table on the page', function() {
+  jasmine.getFixtures().fixturesPath = 'base/specs/fixtures/';
+  loadFixtures('enhanced-tables.html');
```

Comment:
we have a pattern of appending `-fixture` to the fixture filename can you add this like the others in the fixtures folder